apps/web/.storybook/GlobalDecorator.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { addDecorator } from '@storybook/react';
+import '../src/Styles/Global.scss';
+
+const GlobalDecorator = (storyFn) => (
+  <div className="global-wrapper">{storyFn()}</div>
+);
+
+addDecorator(GlobalDecorator);
```

Comment:
Be careful! 

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md?ref=storybookblog.ghost.io#deprecated-addparameters-and-adddecorator

If you hover over `addDecorator` in VSCode does it say that the method is deprecated after v6.0?